apps/desktop/src-tauri/tauri.conf.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://schema.tauri.app/config/2",
   "productName": "Liminal Notes",
   "version": "0.1.0",
-  "identifier": "com.liminal-notes.desktop",
+  "identifier": "ca.liminal.notes",
```

Comment:
For now let's go with my domain `liminal-notes.ca`, convert this to the identifier field, please.